apisix/plugins/cas-auth.lua

```diff
@@ -49,6 +49,8 @@ local _M = {
 }
 
 function _M.check_schema(conf)
+    local check = {"idp_uri"}
```

Comment:
Should `cas_callback_uri` and `logout_uri` be listed as checks? Similar fields in `authz-casdoor` and `authz-keycloak` are listed as checks.

Reply:
cas_callback_uri and logout_uri are in the form of "/foo/bar" so we don't need to check them.